spec/features/visitors/bad_password_spec.rb

```diff
@@ -15,14 +16,28 @@
       expect(page).to have_content(error_message)
       expect(page).to have_current_path(new_user_session_path)
     end
+    locked_out_time_remaining = distance_of_time_in_words(5.minutes)
+    allow_any_instance_of(Users::SessionsController).to receive(:locked_out_time_remaining).
+      and_return(locked_out_time_remaining)
```

Comment:
Adding to previous comment, it looks like we don't have any test coverage for `locked_out_time_remaining`.

I think it's tricky in the context of the feature spec. Unless we have some way to access the actual session to see when the lockout period actually started, maybe it's okay to stub this, as long as we have some coverage somewhere else.

Reply:
Yea I think we had to stub it in the feature text because we freeze time which makes the time methods a little wonky. but will add coverage for locked_out_time_remaining.